bin/verify_boilerplate.py

```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/env python3
```

Comment:
I think this file was meant to be excluded, given the `.gitignore` contents.

Reply:
yep